src/plugins/credential_management/public/components/create_button/create_button.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// @ts-ignore
+import { euiColorAccent } from '@elastic/eui/dist/eui_theme_light.json';
+import React from 'react';
+import * as H from 'history';
```

Comment:
why not just
`import {History} from 'history';`

Reply:
`History` is a reserved keyword

Reply:
hmm, actually `history` is a reserved keyword for window object, `History` should not be, can you confirm?

Reply:
Use `import {History} from 'history';`